peer.go

```diff
@@ -1877,6 +1936,14 @@ out:
 				continue
 			}
 
+			// Determine if the channel is in the activeChannels map and set
+			// the syncStates bool in the call to addLink with this value.
+			// If it's already set, then we need to send ChannelReestablish
+			// on starting up the link.
+			_, ok := p.activeChannels[chanID]
```

Comment:
We already check above that the channel is active, as a result, I think this check here can be skipped.

Reply:
I think it helps for clarity. In the statement above that section with:

```
if currentChan, ok := p.activeChannels[chanID]; ok &&
```

this is *specifically* for checking if the channel has already been loaded into the switch.
In the case you mentioned, I renamed `ok` to `shouldSync` and I think now it would make sense to the reader that this is for telling the link whether to sync states, wyt?